### PR TITLE
Add navbar navigation

### PR DESCRIPTION
**Pull request by alizard0**
Hey 👋🏽 

I mainly added:
1. Navbar navigation (projects, blog, community and about pages)
2. Removed _site folder because there is no need to override this as github pages manages that for us
3. Create two layouts: default and index. So, the idea is to provide some hierarchy in the pages avoiding writing all that html
4. Fixed some minor issues.

Sorry for the bad commit messages but I was forced to kinda used this to test the changes as I could not get jekyll working locally. ( yes, that might be something we should investigate and update the readme)

last notes, I think these changes make the content creation (blog posts) a bit easier as you only need to write a md file inside posts and everything will update automatically.

ah! https://qiot-project.github.io/ is poiting to `add-navbar-template` branch if you want to validate the changes before approving them!

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.7.1 -->
+  <title>QIoT project | QIoT: Quarkus and Internet of Things</title>
+  <meta name="generator" content="Jekyll v4.1.1" />
+  <meta property="og:title" content="QIoT project" />
+  <meta property="og:locale" content="en_US" />
+  <meta name="description" content="QIoT: Quarkus and Internet of Things" />
+  <meta property="og:description" content="QIoT: Quarkus and Internet of Things" />
+  <link rel="canonical" href="http://localhost:4000/" />
```
Comment (eye0fra) [inline _layouts/default.html]: It should be: https://qiot-project.github.io

File: _layouts/default.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <meta property="og:url" content="http://localhost:4000/" />
```
Comment (eye0fra) [inline _layouts/default.html]: Same here: https://qiot-project.github.io

File: _layouts/default.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  <meta property="og:site_name" content="QIoT project" />
+  <meta name="twitter:card" content="summary" />
+  <meta property="twitter:title" content="QIoT project" />
+  <script type="application/ld+json">
+  {"description":"QIoT: Quarkus and Internet of Things","@type":"WebSite","headline":"QIoT project","url":"http://localhost:4000/","name":"QIoT project","@context":"https://schema.org"}</script>
```
Comment (eye0fra) [inline _layouts/default.html]: and here :)

File: _layouts/default.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  <!-- End Jekyll SEO tag -->
+  <link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="QIoT project" /></head>
```
Comment (eye0fra) [inline _layouts/default.html]: Can be the feed.xml relative path?

**Review by eye0fra — CHANGES_REQUESTED** [review]
All great! just small chances. Thanks a lot

File: _layouts/default.html (hunk 4 above)
Comment (alizard0) [reply _layouts/default.html]: No idea !

Comment (alizard0) [reply _layouts/default.html]: It seems, https://qiot-project.github.io/feed.xml does the trick!

**Comment by alizard0** [comment]
@eye0fra just committed the changes you request. Could you please re-review it?

**Review by eye0fra — APPROVED** [review]

**Pull request merged**